Common/include/CConfig.hpp

```diff
@@ -2106,22 +2139,133 @@ class CConfig {
    */
   bool GetSpecies_StrongBC() const { return Species_StrongBC; }
 
+  /*!
+   * \brief Get the flame offset (point on plane separating burnt and unburnt zone)
+            for flamelet model initialization.
+   * \return flame offset for flamelet model initialization
+   */
+  su2double *GetFlameOffset(void) { return flame_offset; }
+
+  /*!
+   * \brief Get the flame normal (of the plane separating burnt and unburnt zone)
+            for flamelet model initialization. The normal points in the direction of the burnt zone.
+   * \return flame offset for flamelet model initialization
+   */
+  su2double *GetFlameNormal(void) { return flame_normal; }
+
+  /*!
+   * \brief Get the flame thickness (reaction zone) for flamelet model initialization.
+            This is the thickness of the transition layer from unburnt to burnt conditions.
+   * \return flame thickness for flamelet model initialization
+   */
+  su2double GetFlameThickness(void) { return flame_thickness; }
+
+  /*!
+   * \brief Get the burnt region thickness for flamelet model initialization.
+            This is the thickness of the hot, burnt zone after the reaction zone.
+   * \return flame thickness for flamelet model initialization
+   */
+  su2double GetFlameBurntThickness(void) { return flame_burnt_thickness; }
+
```

Comment:
Please const everything you can
```suggestion
  su2double GetFlameThickness(void) const { return flame_thickness; }

  /*!
   * \brief Get the burnt region thickness for flamelet model initialization.
            This is the thickness of the hot, burnt zone after the reaction zone.
   * \return flame thickness for flamelet model initialization
   */
  su2double GetFlameBurntThickness(void) const { return flame_burnt_thickness; }

```